databases/week2/main3.sql

```diff
@@ -0,0 +1,53 @@
+--Part 3: More queries
+SET
+  NAMES utf8mb4;
+USE hyfdbweek2;
+--1.Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT
+  task.title,
+  user.email
+FROM
+  task
```

Comment:
formatting tip, would always keep from + table name on the same line. 
Pointer: improves readability and clarity

Reply:
see other occurrences and  update